src/ci/agent.sh

```diff
@@ -34,6 +34,12 @@ if [ X${CARGO_INCREMENTAL} == X ]; then
     cargo clean
 fi
 
+rustc --version
```

Comment:
Maybe move this to just _above_ the first `cargo` invocation. That way, for example, if `cargo clean` ever failed inconsistently, we could compare tool versions between runs.

Reply:
Do you mean the whole block of `--version` or just the rustc one?

Reply:
I'd say all of them (particularly `cargo` per se), just to keep it simple!